discovery-provider/src/tasks/index.py

```diff
@@ -328,10 +354,12 @@ def index_blocks(self, db, blocks_list):
             if playlist_state_changed:
                 if playlist_ids:
                     remove_cached_playlist_ids(redis, playlist_ids)
+            logger.info(f"index.py | redis cache clean operations complete")
 
         # add the block number of the most recently processed block to redis
         redis.set(most_recent_indexed_block_redis_key, block.number)
         redis.set(most_recent_indexed_block_hash_redis_key, block.hash.hex())
+        logger.info(f"index.py | update most recently processed block complete")
```

Comment:
i don't think we should have logs that don't contain anything - let's embed the most recently processed block in this msg/